cmd/utils/flags.go

```diff
@@ -112,3 +141,7 @@ func AddNodeFlags(command *cobra.Command, vm *VirtualMachineScaleSetVM) {
 		return nil
 	}
 }
+
+func ConfigFile() string {
+	return path.Join(cacheDir(), "config.yaml")
```

Comment:
IMO, `cacheDir` should become `configDir` (It will still be `$HOME/.kubectl-az`) because it now contains the cache file but also the configuration file.